tests/functional/storageclass/test_cross_sc_clone_snap_restore.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+import logging
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework.testlib import ManageTest, tier2
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from tests.fixtures import create_project
+from ocs_ci.ocs.resources import pvc
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
+@tier2
+@pytest.mark.usefixtures(
+    create_project.__name__,
+)
+class TestCrossScCloneSnapRestore(ManageTest):
+    """
+    The test verifies creation of clones and restore of snapshots to other than pvc storage class
+    """
+
+    @pytest.mark.parametrize(
+        argnames="interface_type",
+        argvalues=[
+            pytest.param(*[constants.CEPHBLOCKPOOL]),
+            pytest.param(*[constants.CEPHFILESYSTEM]),
+        ],
+    )
+    def test_cross_class_same_pool_clone_snap_restore(
+        self,
+        interface_type,
+        storageclass_factory,
+        pvc_factory,
+        pod_factory,
+        pvc_clone_factory,
+        snapshot_factory,
+    ):
+        """
+        Test function which validates operations accross different storage classes.
+        1. Create two storage classes in the same pool
+        2. Create a pvc in the first sc
+        3. Create a clone on the same sc
+        4. Create a clone on another sc
+        5. Create pvc's shapshot
+        6. Restore the snapshot on the same sc
+        7. Restore the snapshot on another sc
+        """
+
+        # Create a Storage Class
+        sc_obj = storageclass_factory(interface=interface_type)
```

Comment:
good to follow similar naming convention. I mean could you change this to sc_obj1 which would be good since there is sc_obj2 in code. having it common might confuse later.

Reply:
Fixed. Thanks.